azurerm/internal/services/azurestackhci/tests/hci_cluster_resource_test.go

```diff
@@ -0,0 +1,228 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/azurestackhci/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMHCICluster_basic(t *testing.T) {
```

Comment:
```suggestion
func TestAccHCICluster_basic(t *testing.T) {
```

Lets remove AzureRM from the test names

Reply:
updated